src/multibody/joint/joint-composite.hpp

```diff
@@ -200,9 +200,11 @@ namespace pinocchio
     /// \param jmodel Model of the joint to add.
     /// \param placement Placement of the joint relatively to its predecessor.
     ///
+    /// \return A reference to this JointModelComposite itself
+    ///
     template<typename JointModel>
-    void addJoint(const JointModelBase<JointModel> & jmodel,
-                  const SE3 & placement = SE3::Identity())
+    JointModelDerived& addJoint(const JointModelBase<JointModel> & jmodel,
```

Comment:
`JointModelDerived &` is more readable I think. And it follows your other modifications.

Reply:
I don't know if it is more readable or not, but I am willing to sick to any coding convention we might have, so I fixed it